### ISSUE TITLE
Update the WCAG understanding links from 2.0 to latest

### ISSUE THREAD

**Issue by kengdoj** [atbcb/ICTTestingBaseline#344]
Discussed 3/15/22. Seems like a good idea, but need to do more research.

**Comment by bruce-usab** [comment]
My understanding is that updated links and consolidation of 2.0/2.1/2.2 versions is planned to coincide with 2.2 final publication of WCAG 2.2.  I believe the idea is (1) Informative materials be moved from w3.org/TR to w3.org/WAI; (2) the URL just be WCAG or WCAG2 and not version specific; and (3) moved to a Netlify CMS so easier to both read and update.  Examples (of new formatting, but not <q>evergreen</q> URLs:
- https://www.w3.org/WAI/WCAG20/Understanding/info-and-relationships
- https://www.w3.org/WAI/WCAG20/Techniques/html/H51

I am sorry to say I am not clear at all what are best URLs to use for now.  Probably ones with <q>21</q> would be most likely to be redirected to latest available.
- https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
- https://www.w3.org/WAI/WCAG21/Techniques/html/H51.html

These also work: 
- https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships
- https://www.w3.org/WAI/WCAG22/Techniques/html/H51

**Comment by kengdoj** [comment]
If the 2.0 Understandings and Techniques will be moved, links throughout the Baseline will need to be updated. 

@bruce-usab - do you know where can we see the changes in the 22 Understanding and Techniques compared to 21 or 20?

**Comment by bruce-usab** [comment]
I know that there have been concerns raised by AG WG members and others that in-process edits are really hard to surface.  The difficulty comes up somewhat routinely in [WCAG/issue](https://github.com/w3c/wcag/issues) threads.  The other confounding factor is that some edits &mdash; which the AG WG _have_ approved &mdash; are still not pulled in to the [current Editor's draft](https://w3c.github.io/wcag/guidelines/22/).

**Comment by bruce-usab** [comment]
This is very manual, but it might work well enough since there is not much of 2.0 which is changing.  Everything is public facing on GitHub.  Here is what I did, for example, to confirm that the number for Relative Luminance are updated;

1. Browse to the '20' subfolder, look for any files which have been changed in the last year:
2. [wcag -> guidelines -> terms -> 20](https://github.com/w3c/wcag/tree/main/guidelines/terms/20
3. [relative luminace](https://github.com/w3c/wcag/blob/main/guidelines/terms/20/relative-luminance.html)

21 folder is new for 2.1.  22 folder is new-in-progress for 2.2

There is also a [manually curated description](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#changes-from-wcag-21-to-wcag-22)

Finally, I feel like [diff](https://git-scm.com/docs/git-diff) should be able to do this for us.

**Comment by bruce-usab** [comment]
Here is a [recent issue / PR thread](https://github.com/w3c/wcag/issues/2242) noting an update to 2.1/2.0 which slipped through the cracks.  Michael C notes that he needs to <q>come up with a reliable way to not miss them in the future</q>.  New tools and procedures, and a steep learning curve.

**Comment by bruce-usab** [comment]
Here is the PR for [Add redesign of Techniques, Understanding templates](https://github.com/w3c/wcag/pull/2012) which describes the good direction WAI is moving to support AGWG and community groups.

**Comment by kengdoj** [comment]
Conforming Alternate Version for Trusted Tester rewrite discussion identified that [2.1 version of Understanding CAV](https://www.w3.org/WAI/WCAG21/Understanding/conformance#conforming-alt-versions) has author requirement to identify CAV:

> Authors relying on conforming alternate versions must make end users aware that a conforming alternate version is available. This may be accomplished by providing a link to a more accessible version, identified clearly by link text. Alternatively a link to instructions may be provided which documents how to access a more accessible version as well as the specific ways the alternate version is more accessible (e.g. a "high contrast version").

[2.0 version of Understanding CAV](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-conforming-alt-versions-head) does not have this.

It was agreed to change links of Understanding to version 2.1 (or 2.2) since these will likely provide better explanations of the SCs. Will review 2.2 versions when available.

**Comment by kengdoj** [comment]
Provide explanation of why not using the 2.0 version: understanding x.x provides additional clarification of the WCAG 2.0 requirements; no change to the requirement is made.

**Comment by kengdoj** [comment]
[2.2 draft definition for CAV](https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions) is the same as previous versions. It points to the [2.1 Understanding CAV](https://www.w3.org/WAI/WCAG21/Understanding/conformance#conforming-alt-versions). 

No 2.2 Understanding is available yet.

**Comment by kengdoj** [comment]
[All WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by kengdoj** [comment]
Make these updates to Appendix C also.

**Comment by kengdoj** [comment]
[2.2 Understanding CAV](https://www.w3.org/WAI/WCAG22/Understanding/conformance#conforming-alt-versions)
